draft thank you 2020 post (#101)
ropensci/roweb3: content/blog/2020-12-15-thank-you-2020/index.md

---
title: Thank You, 2020
author:
  - Stefanie Butland
  - Steffi LaZerte
date: '2020-12-15'
slug: thankyou-2020
tags:
  - thank you
  - community

> description: A very short summary of your post (~ 100 characters)

Reviewer: Update this
```suggestion
description: Thank you rOpenSci Community!
```

On behalf of rOpenSci, thank you. 

People have submitted their R packages for software peer review (16 package authors)[^1], reviewed those packages (**~XX** reviewers), contributed code or documentation (84 people made their first code contribution to rOpenSci this year), authored a [blog post or tech note](/archive/) (43 community member authors, including 21 first-time authors), shared a [use case](https://discuss.ropensci.org/c/usecases/) to help package authors see how their work is being used and help other users imagine how they can apply it (42 people), attended a [Community Call](/commcalls/) (136 people in 19 countries), cited our software (380 citations of 132 packages), asked or answered questions, commented on project ideas, added notes to a public doc, or gave us a shoutout in a talk, a post, or on Twitter.

Some people’s volunteer contributions this year warrant a special mention:

> *   Volunteer Editors for Software Peer Review, [Brooke Anderson](/author/brooke-anderson/), [Anna Krystalli](/author/anna-krystalli/), [Laura DeCicco](/author/laura-decicco/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Noam Ross](/author/noam-ross/), [Melina Vidoni](/author/melina-vidoni/), and [Lincoln Mullen](/author/lincoln-mullen/), now Editor alum, guest editor [Ana Laura Diedrichs](https://github.com/anadiedrichs), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/), and [Scott Chamberlain](/author/scott-chamberlain/), now Editor alum;

Reviewer 2: ```suggestion
*   Volunteer Editors for Software Peer Review, [Brooke Anderson](/author/brooke-anderson/), [Anna Krystalli](/author/anna-krystalli/), [Laura DeCicco](/author/laura-decicco/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Noam Ross](/author/noam-ross/), [Melina Vidoni](/author/melina-vidoni/), and [Lincoln Mullen](/author/lincoln-mullen/), now Editor alum, guest editor [Ana Laura Diedrichs](https://github.com/anadiedrichs), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/), and [Scott Chamberlain](/author/scott-chamberlain/), now Editor alumnus;
```

I didn't know the abbreviation "alum"

> *   Volunteer Editors for Software Peer Review, [Brooke Anderson](/author/brooke-anderson/), [Anna Krystalli](/author/anna-krystalli/), [Laura DeCicco](/author/laura-decicco/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Noam Ross](/author/noam-ross/), [Melina Vidoni](/author/melina-vidoni/), and [Lincoln Mullen](/author/lincoln-mullen/), now Editor alum, guest editor [Ana Laura Diedrichs](https://github.com/anadiedrichs), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/), and [Scott Chamberlain](/author/scott-chamberlain/), now Editor alumnus;

Author: ```suggestion
*   Volunteer Editors for Software Peer Review, [Brooke Anderson](/author/brooke-anderson/), [Anna Krystalli](/author/anna-krystalli/), [Laura DeCicco](/author/laura-decicco/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Noam Ross](/author/noam-ross/), [Melina Vidoni](/author/melina-vidoni/), and [Lincoln Mullen](/author/lincoln-mullen/), now Editor alumnus, guest editor [Ana Laura Diedrichs](https://github.com/anadiedrichs), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/), and [Scott Chamberlain](/author/scott-chamberlain/), now Editor alumnus;
```

*   Steering Committee members for our nascent Statistical Software Peer Review, [Ben Bolker](https://ms.mcmaster.ca/~bolker/), [Leonardo Collado-Torres](http://lcolladotor.github.io/), [Stephanie Hicks](https://www.stephaniehicks.com/), [Rebecca Killick](http://www.lancs.ac.uk/~killick/), [Max Kuhn](http://appliedpredictivemodeling.com/), [Paula Moraga](http://www.paulamoraga.com/);
*   New maintainers of existing rOpenSci packages, [Stéphane Guillou](https://stragu.gitlab.io/), [Sergio Ibarra-Espinosa](https://github.com/ibarraespinosa), [Amanda Rehbein](https://github.com/salvatirehbein);
*   Presenters in our Community Calls, [Carl Boettiger](/author/carl-boettiger/), [Scott Chamberlain](/author/scott-chamberlain/), [Leonardo Collado-Torres](/author/leonardo-collado-torres/), [Erin Grand](/author/erin-grand/), [Matt Jones](/author/matt-jones/), [Daniella Lowenberg](/author/daniella-lowenberg/), [Karthik Ram](/author/karthik-ram/), [Julia Silge](/author/julia-silge/), [Elin Waring](/author/elin-waring/), [Kara Woo](/author/kara-woo/);

> *   Community members who inspired us to quote them in the rOpenSci [Community Contributing Guide](https://contributing.ropensci.org/)[^2], [Shannon Ellis](/author/shannon-e.-ellis/), [Miles McBain](/author/miles-mcbain/), [Janani Ravi](/author/janani-ravi/), [Eric Scott](https://www.ericrscott.com/), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/), [Jennifer Thompson](/author/jennifer-thompson/), and those who reviewed parts of the Guide, [Toby Hodges](https://github.com/tobyhodges), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/);

Author: ```suggestion
*   Community members who inspired us to quote them in the rOpenSci [Community Contributing Guide](https://contributing.ropensci.org/)[^2], [Shannon Ellis](/author/shannon-e.-ellis/), [Miles McBain](/author/miles-mcbain/), [Janani Ravi](/author/janani-ravi/), [Eric Scott](https://www.ericrscott.com/), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/), [Jennifer Thompson](/author/jennifer-thompson/), and those who reviewed parts of the guide, [Toby Hodges](https://github.com/tobyhodges), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/);
```

> *   Contributors to our guides, rOpenSci Statistical Software Peer Review[^3], **Mark do I add [all Content contributors](https://ropenscilabs.github.io/statistical-software-review-book/index.html#contributors) but not Issues Authors?**, rOpenSci Packages: Development, Maintenance, and Peer Review[^4], **Maëlle pls add dev guide 2020 contributors**, and rOpenSci Community Contributing Guide[^2], [Toby Hodges](https://github.com/tobyhodges), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/);

Reviewer 2: ```suggestion
*   Contributors to our guides, rOpenSci Statistical Software Peer Review[^3], **Mark do I add [all Content contributors](https://ropenscilabs.github.io/statistical-software-review-book/index.html#contributors) but not Issues Authors?**, rOpenSci Packages: Development, Maintenance, and Peer Review[^4]; [Aaron Wolen](https://github.com/aaronwolen), [Hugo Gruson](https://github.com/Bisaloo), [Luke McGuinness](https://github.com/mcguinlu), and rOpenSci Community Contributing Guide[^2], [Toby Hodges](https://github.com/tobyhodges), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/);
```

Reviewer 2: other contributors are staff members (you, Jeroen, Mark), editors, leadership members (Carl) so not noted. I am a bit surprised but I think it's because I feel like the dev guide is absorbing wisdom from the whole community which direct issues/PRs doesn't reflect?

Reviewer 2: should there be a link to https://devguide.ropensci.org/booknews.html btw?

Author: I decided not to include that NEWS link but added a line about "community wisdom". Thank you!

That bullet needs editing but content is good now

> *   Contributors to our guides, rOpenSci Statistical Software Peer Review[^3], **Mark do I add [all Content contributors](https://ropenscilabs.github.io/statistical-software-review-book/index.html#contributors) but not Issues Authors?**, rOpenSci Packages: Development, Maintenance, and Peer Review[^4]; [Aaron Wolen](https://github.com/aaronwolen), [Hugo Gruson](https://github.com/Bisaloo), [Luke McGuinness](https://github.com/mcguinlu), and rOpenSci Community Contributing Guide[^2], [Toby Hodges](https://github.com/tobyhodges), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/);

Author: ```suggestion
*   Contributors to our guides, rOpenSci Statistical Software Peer Review[^3], **Mark do I add [all Content contributors](https://ropenscilabs.github.io/statistical-software-review-book/index.html#contributors) but not Issues Authors?**, rOpenSci Packages: Development, Maintenance, and Peer Review[^4], [Carl Boettiger](/author/carl-boettiger/), [Hugo Gruson](/author/hugo-gruson/), [Luke McGuinness](/author/luke-mcguinness/), [Aaron Wolen](/author/aaron-wolen/), Editors, and rOpenSci Community Contributing Guide[^2], [Toby Hodges](https://github.com/tobyhodges), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/), plus wisdom from the whole community that isn't reflected in direct issues and pull requests;
```

---
title: Thank You, 2020
author:
  - Stefanie Butland
  - Steffi LaZerte
date: '2020-12-15'
slug: thankyou-2020
tags:
  - thank you
  - community
description: Thank you rOpenSci Community!
# twitterImg: blog/2019/06/04/post-template/name-of-image.png
# twitterAlt: Alternative description of the image
---

> In a year where it has been hard to pay attention to anything not critical to our day-to-day lives, you have continued to share your time, expertise, enthusiasm, and willingness to try things with us. Our [staff](/about/#team) of developers, researchers, and community builders work to build technical and social infrastructure to lower barriers to working with research data, and our community continually helps us push farther.

Author: ```suggestion
In a year where it has been hard to pay attention to anything not critical to our day-to-day lives, you have continued to share your time, expertise, enthusiasm, and willingness to try things with us. Our [staff](/about/#team) of developers, researchers, and community builders work to build technical and social infrastructure to lower barriers to working with research data, and you, our community, continually help us push farther.
```

> In a year where it has been hard to pay attention to anything not critical to our day-to-day lives. you have continued to share your time, expertise, enthusiasm, and willingness to try things with us. Our [staff team](/about/#team) of developers, researchers, and community builders work to build technical and social infrastructure to lower barriers to working with research data, and you, our community, continually help us push farther than we thought we could go.

Author: ```suggestion
In a year where it has been hard to pay attention to anything not critical to our day-to-day lives, you have continued to share your time, expertise, enthusiasm, and willingness to try things with us. Our [staff](/about/#team) of developers, researchers, and community builders work to build technical and social infrastructure to lower barriers to working with research data, and our community continually helps us push farther.
```

On behalf of rOpenSci, thank you. 

People have submitted their R packages for software peer review (16 package authors)[^1], reviewed those packages (**~XX** reviewers), contributed code or documentation (84 people made their first code contribution to rOpenSci this year), authored a [blog post or tech note](/archive/) (43 community member authors, including 21 first-time authors), shared a [use case](https://discuss.ropensci.org/c/usecases/) to help package authors see how their work is being used and help other users imagine how they can apply it (42 people), attended a [Community Call](/commcalls/) (136 people in 19 countries), cited our software (380 citations of 132 packages), asked or answered questions, commented on project ideas, added notes to a public doc, or gave us a shoutout in a talk, a post, or on Twitter.

Some people’s volunteer contributions this year warrant a special mention:

> *   Volunteer Editors for Software Peer Review, [Brooke Anderson](/author/brooke-anderson/), [Anna Krystalli](/author/anna-krystalli/), [Laura DeCicco](/author/laura-decicco/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Noam Ross](/author/noam-ross/), [Melina Vidoni](/author/melina-vidoni/), [Lincoln Mullen](/author/lincoln-mullen/) (now Editor alumnus), guest editor [Ana Laura Diedrichs](https://github.com/anadiedrichs), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/), [Scott Chamberlain](/author/scott-chamberlain/) (now Editor alumnus);

Author: ```suggestion
*   Volunteer Editors for Software Peer Review, [Brooke Anderson](/author/brooke-anderson/), [Anna Krystalli](/author/anna-krystalli/), [Laura DeCicco](/author/laura-decicco/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Noam Ross](/author/noam-ross/), [Melina Vidoni](/author/melina-vidoni/), [Lincoln Mullen](/author/lincoln-mullen/) (Editor alumnus), guest editor [Ana Laura Diedrichs](https://github.com/anadiedrichs), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/), [Scott Chamberlain](/author/scott-chamberlain/) (Editor alumnus);
```

---

> title: Thank You, 2020

Reviewer 2: I know why it's called like that but this year it sounds a bit weird. How about something longer like "Thank You rOpenSci Community for Your Involvement in 2020"?

author:
  - Stefanie Butland
  - Steffi LaZerte
date: '2020-12-15'
slug: thankyou-2020
tags:
  - thank you
  - community
description: Thank you rOpenSci Community!
# twitterImg: blog/2019/06/04/post-template/name-of-image.png
# twitterAlt: Alternative description of the image
---

> In a year where it has been hard to pay attention to anything not critical to our day-to-day lives, you have continued to share your time, expertise, enthusiasm, and willingness to try things with us. Our [staff](/about/#team) of developers, researchers, and community builders work to build technical and social infrastructure to lower barriers to working with research data, and you, our community, continually help us push farther.

Reviewer 2: ```suggestion
In a year where it has been hard to pay attention to anything not critical to our day-to-day lives, you have continued to share your time, expertise, enthusiasm, and willingness to try things with us. Our [staff](/about/#team) of developers, researchers, and community builders work to assemble technical and social infrastructure to lower barriers to working with research data, and you, our community, continually help us push farther.
```

Not very happy with my suggestion either, but the repetition builders/build caught my eye when reading.

On behalf of rOpenSci, thank you. 

> People have submitted their R packages for software peer review (16 package authors)[^1], reviewed those packages (31 reviewers[^2]), contributed code or documentation (84 people made their first code contribution to rOpenSci this year), authored a [blog post or tech note](/archive/) (43 community member authors, including 21 first-time authors), shared a [use case](https://discuss.ropensci.org/c/usecases/) to help package authors see how their work is being used and help other users imagine how they can apply it (42 people), attended a [Community Call](/commcalls/) (136 people in 19 countries), cited our software (380 citations of 132 packages), asked or answered questions, commented on project ideas, added notes to a public doc, or gave us a shoutout in a talk, a post, or on Twitter.

Reviewer 2: I'll update the use cases JSON. Could we link to the website use cases webpage instead of the forum?

Reviewer 2: For citations could you link to /citations/?

> People have submitted their R packages for software peer review (16 package authors)[^1], reviewed those packages (31 reviewers[^2]), contributed code or documentation (84 people made their first code contribution to rOpenSci this year), authored a [blog post or tech note](/archive/) (43 community member authors, including 21 first-time authors), shared a [use case](https://discuss.ropensci.org/c/usecases/) to help package authors see how their work is being used and help other users imagine how they can apply it (42 people), attended a [Community Call](/commcalls/) (136 people in 19 countries), cited our software (380 citations of 132 packages), asked or answered questions, commented on project ideas, added notes to a public doc, or gave us a shoutout in a talk, a post, or on Twitter.

Author: ```suggestion
People have submitted their R packages for software peer review (16 package authors)[^1], reviewed those packages (31 reviewers[^2]), contributed code or documentation (84 people made their first code contribution to rOpenSci this year), authored a [blog post or tech note](/archive/) (43 community member authors, including 21 first-time authors), shared a [use case](https://discuss.ropensci.org/c/usecases/) to help package authors see how their work is being used and help other users imagine how they can apply it (42 people), attended a [Community Call](/commcalls/) (136 people in 19 countries), cited our software (380 [citations](/citations/) of 132 packages), asked or answered questions, commented on project ideas, added notes to a public doc, or gave us a shoutout in a talk, a post, or on Twitter.
```

> People have submitted their R packages for software peer review (16 package authors)[^1], reviewed those packages (31 reviewers[^2]), contributed code or documentation (84 people made their first code contribution to rOpenSci this year), authored a [blog post or tech note](/archive/) (43 community member authors, including 21 first-time authors), shared a [use case](https://discuss.ropensci.org/c/usecases/) to help package authors see how their work is being used and help other users imagine how they can apply it (42 people), attended a [Community Call](/commcalls/) (136 people in 19 countries), cited our software (380 [citations](/citations/) of 132 packages), asked or answered questions, commented on project ideas, added notes to a public doc, or gave us a shoutout in a talk, a post, or on Twitter.

Author: ```suggestion
People have submitted their R packages for software peer review (16 package authors)[^1], reviewed those packages (31 reviewers[^2]), contributed code or documentation (84 people made their first code contribution to rOpenSci this year), authored a [blog post or tech note](/archive/) (43 community member authors, including 21 first-time authors), shared a [use case](https://discuss.ropensci.org/c/usecases/) to help package authors see how their work is being used and help other users imagine how they can apply it (42 people), attended a [Community Call](/commcalls/) (136 people in 19 countries), cited our software (380 [citations](/citations/) of 132 packages), asked or answered questions, commented on project ideas, added notes to a public doc, invited us to talk about our work, or gave us a shoutout in a presentation, a post, or on Twitter.
```

Some people’s volunteer contributions this year warrant a special mention:

> *   Volunteer Editors for Software Peer Review, [Brooke Anderson](/author/brooke-anderson/), [Anna Krystalli](/author/anna-krystalli/), [Laura DeCicco](/author/laura-decicco/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Noam Ross](/author/noam-ross/), [Melina Vidoni](/author/melina-vidoni/), [Lincoln Mullen](/author/lincoln-mullen/) (Editor alumnus), guest editor [Ana Laura Diedrichs](https://github.com/anadiedrichs), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/), [Scott Chamberlain](/author/scott-chamberlain/) (Editor alumnus);

Reviewer 2: link to /software-review/?

> *   Steering Committee members for our nascent Statistical Software Peer Review, [Ben Bolker](https://ms.mcmaster.ca/~bolker/), [Leonardo Collado-Torres](http://lcolladotor.github.io/), [Stephanie Hicks](https://www.stephaniehicks.com/), [Rebecca Killick](http://www.lancs.ac.uk/~killick/), [Max Kuhn](http://appliedpredictivemodeling.com/), [Paula Moraga](http://www.paulamoraga.com/);

Reviewer 2: link to /stat-software-review/?

*   Presenters in our Community Calls, [Carl Boettiger](/author/carl-boettiger/), [Scott Chamberlain](/author/scott-chamberlain/), [Leonardo Collado-Torres](/author/leonardo-collado-torres/), [Erin Grand](/author/erin-grand/), [Matt Jones](/author/matt-jones/), [Daniella Lowenberg](/author/daniella-lowenberg/), [Karthik Ram](/author/karthik-ram/), [Julia Silge](/author/julia-silge/), [Elin Waring](/author/elin-waring/), [Kara Woo](/author/kara-woo/);
*   Community members who inspired us to quote them in the rOpenSci [Community Contributing Guide](https://contributing.ropensci.org/)[^3], [Shannon Ellis](/author/shannon-e.-ellis/), [Miles McBain](/author/miles-mcbain/), [Janani Ravi](/author/janani-ravi/), [Eric Scott](https://www.ericrscott.com/), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/), [Jennifer Thompson](/author/jennifer-thompson/), and those who reviewed parts of the guide, [Toby Hodges](https://github.com/tobyhodges), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/);
*   Contributors to our guides

>     *   rOpenSci Statistical Software Peer Review[^4], **Mark do I add [all Content contributors](https://ropenscilabs.github.io/statistical-software-review-book/index.html#contributors) but not Issues Authors?**

Reviewer 2: why are the links to guides footnotes?

Reviewer 2: I can see that it means they are references, but I'd find it easier to have actual links (also for the footnote linking a blog post) because I can hover on it to see what it is as opposed to clicking, seeing the link, then clicking on the link back.

    *  Plus wisdom from the whole community that isn't reflected in direct issues and pull requests to specific guides
*   Authors of our special covid-era post on Leveraging Open Data Resources for Remote Research[^6], [Sam Albers](/author/sam-albers/), [Brooke Anderson](/author/brooke-anderson/), [Robin Lovelace](/author/robin-lovelace/), [Ben Marwick](/author/ben-marwick/), [Anton Van de Putte](/author/anton-van-de-putte/), [Ilaria Prosdocimi](/author/ilaria-prosdocimi/), [Ben Raymond](/author/ben-raymond/), [Louise Slater](/author/louise-slater/), [Claudia Vitolo](/author/claudia-vitolo/), [Sam Zipper](/author/sam-zipper/);
*   Captioners for a Community Call who wrote up what they learned[^7], [Matthias Grenié](/author/matthias-grenié/), [Hugo Gruson](/author/hugo-gruson/);
*   External member of our Code of Conduct Committee, [Kara Woo](/author/kara-woo/);
*   rOpenSci Strategic Advisors, [Carl Boettiger](/author/carl-boettiger/), 
[Jenny Bryan](/author/jenny-bryan/), [Rich FitzJohn](/author/rich-fitzjohn/).

Thank you **all** for making rOpenSci.

> {{< figure src = "pets.png" class = "center" width = "400px" caption = "Nami (Steffi's Assistant) and Ruby (Stefanie's Assistant)" alt = "grey cat and black dog above the words Thank you" >}}

Reviewer 2: this is the best part of the post!! :blue_heart:

*   Presenters in our Community Calls, [Carl Boettiger](/author/carl-boettiger/), [Scott Chamberlain](/author/scott-chamberlain/), [Leonardo Collado-Torres](/author/leonardo-collado-torres/), [Erin Grand](/author/erin-grand/), [Matt Jones](/author/matt-jones/), [Daniella Lowenberg](/author/daniella-lowenberg/), [Karthik Ram](/author/karthik-ram/), [Julia Silge](/author/julia-silge/), [Elin Waring](/author/elin-waring/), [Kara Woo](/author/kara-woo/);
*   Community members who inspired us to quote them in the rOpenSci [Community Contributing Guide](https://contributing.ropensci.org/)[^2], [Shannon Ellis](/author/shannon-e.-ellis/), [Miles McBain](/author/miles-mcbain/), [Janani Ravi](/author/janani-ravi/), [Eric Scott](https://www.ericrscott.com/), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/), [Jennifer Thompson](/author/jennifer-thompson/), and those who reviewed parts of the guide, [Toby Hodges](https://github.com/tobyhodges), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/);

> *   Contributors to our guides, rOpenSci Statistical Software Peer Review[^3], **Mark do I add [all Content contributors](https://ropenscilabs.github.io/statistical-software-review-book/index.html#contributors) but not Issues Authors?**, rOpenSci Packages: Development, Maintenance, and Peer Review[^4], [Carl Boettiger](/author/carl-boettiger/), [Hugo Gruson](/author/hugo-gruson/), [Luke McGuinness](/author/luke-mcguinness/), [Aaron Wolen](/author/aaron-wolen/), Editors, and rOpenSci Community Contributing Guide[^2], [Toby Hodges](https://github.com/tobyhodges), [Julia Stewart Lowndes](/author/julia-stewart-lowndes/), plus wisdom from the whole community that isn't reflected in direct issues and pull requests;

Reviewer 2: I love this phrasing!!
